tests/slack_sdk_async/web/test_web_client_coverage.py

```diff
@@ -366,6 +366,23 @@ async def run_method(self, method_name, method, async_method):
                     method(client_id="111.222", client_secret="xxx")["method"]
                 )
                 await async_method(client_id="111.222", client_secret="xxx")
+            elif method_name == "apps_manifest_create":
```

Comment:
I've always had issues with running tests locally, but the ones currently failing either are `websockets` or `sqlite3` related. Can debug further tomorrow, but would appreciate any guidance on how to ensure all is well.

Reply:
For the sqlite3 errors, perhaps your local environment has an old version of file based databases. You can delete `logs/*.db` files to fix the errors. I'm not sure about the error with websockets package but you can try running tests using the scripts under `scripts` directory after setting up a plain python venv (`python 3 -m venv .venv && source .venv/bin/activate`).